.travis.yml

```diff
@@ -30,12 +30,29 @@ before_install:
   # Install symfony/error-handler on compatible PHP versions to avoid a deprecation warning of the old DebugClassLoader and ErrorHandler classes
   - if [[ "$TRAVIS_PHP_VERSION" != 5.* && "$TRAVIS_PHP_VERSION" != 7.0 && "$SYMFONY_VERSION" = "" ]]; then composer require --no-update --dev symfony/error-handler "^4.4 || ^5.0"; fi;
   - sh -c 'if [ "$SYMFONY_VERSION" != "" ]; then composer require -n --no-update symfony/lts=$SYMFONY_VERSION; fi;'
+  - if (( "${TRAVIS_PHP_VERSION:0:1}" >= "8" )) ; then composer require --dev --no-update  "phpunit/phpunit:^7.5.18"; fi;
 
 install:
-  - composer install -n --prefer-dist
-
-script: vendor/bin/phpunit -v --coverage-clover=coverage.clover
+  -  if (( "${TRAVIS_PHP_VERSION:3}" == "snapshot" )) ; then composer install --ignore-platform-req=php -n --prefer-dist; else  composer install  -n --prefer-dist; fi;
+
+script: |
+  #!/usr/bin/env bash
+  set -e
```

Comment:
This is breaking Travis itself. Scripts put in the travis.yml file are embedded in the generated build script, so this turns on `-e` for the Travis build script itself.

Either don't use it, or put your bash script as a dedicated file that you can call here (so that `set -e` only affects it)

Reply:
Done